src/Components/Sidebar/Sidebar.js

```diff
@@ -14,12 +14,44 @@ export default function Sidebar() {
       navigate("/");
     });
   }, [navigate]);
+  const optionsDemo = [
+    { name: "weekly progress", id: "abc" },
+    { name: "cool talks", id: "def" },
+    { name: "In the middle of june", id: "ghi" },
+  ];
+  const [menuOptions, setMenuOptions] = useState(optionsDemo);
+  console.log(setMenuOptions);
+  // will set it once api starts working
+  const sidebarMenuOptions = menuOptions.map((item, key) => {
+    return (
+      <div
+        className="sidebar__menu--options"
+        key={key}
+        onClick={() => console.log(`backend/document/${item.id} is clicked`)}
+        // will convert it to another function and add useCallback
+      >
+        {item.name}
+      </div>
+    );
+  });
+
+  const SidebarMenu = useCallback(() => {
```

Comment:
why captial S?

Reply:
also functions are better named with the action they do; like openSideBar, closeSidebar, killManthan etc.

Reply:
okaysir